esmvalcore/config-developer.yml

```diff
@@ -203,7 +208,11 @@ obs4mips:
   input_dir:
     default: 'Tier{tier}/{dataset}'
     RCAST: '/'
-  input_file: '{short_name}_{dataset}_{level}_{version}_*.nc'
+    IPSL: '{realm}/{short_name}/{freq}/{grid}/{institute}/${dataset}/{latest_version}'
+  input_file:
+    default: '{short_name}_{dataset}_{level}_{version}_*.nc'
+    RCAST: '{short_name}_{dataset}_{level}_{version}_*.nc'
```

Comment:
this looks the same as the `default`, does it need to be added?

Reply:
No, you're right. Done